pkg/ddevapp/app_compose_template.yaml

```
@@ -182,6 +182,7 @@ services:
     - COREPACK_ENABLE_DOWNLOAD_PROMPT=0
     - COREPACK_HOME=/mnt/ddev-global-cache/corepack
     - DOCROOT=${DDEV_DOCROOT}
+    - DDEV_APPROOT="/var/www/html"
```

```
$ ddev exec 'echo $DDEV_APPROOT/test'
"/var/www/html"/test
```

But I expected to see it without quotes.

```suggestion
    - DDEV_APPROOT=/var/www/html
```